backend-lib/src/main/rust/lib.rs

```diff
@@ -1173,7 +1173,7 @@ pub unsafe extern "C" fn Java_cash_z_ecc_android_sdk_internal_jni_RustBackend_ge
         match db_data
             .get_wallet_summary(0)
             .map_err(|e| format_err!("Error while fetching scan progress: {}", e))?
-            .and_then(|summary| summary.scan_progress().filter(|r| r.denominator() > 0))
+            .and_then(|summary| summary.scan_progress().filter(|r| r.denominator() > &0))
```

Comment:
@nuttycom Please review this Rust FFI change, as I just used what the compiler suggested to silent error:
```
error[E0308]: mismatched types
    --> src/main/rust/lib.rs:1176:86
     |
1176 |             .and_then(|summary| summary.scan_progress().filter(|r| r.denominator() > 0))
     |                                                                                      ^
     |                                                                                      |
     |                                                                                      expected `&u64`, found integer
     |                                                                                      help: consider borrowing here: `&0`
```

Reply:
Yup, sorry I missed that.